allowed-breaking-changes.txt

```diff
@@ -19,3 +19,6 @@ changed-type:@aws-cdk/aws-dynamodb.Table.tableStreamArn
 incompatible-argument:@aws-cdk/aws-apigateway.LambdaRestApi.addModel
 incompatible-argument:@aws-cdk/aws-apigateway.Model.<initializer>
 incompatible-argument:@aws-cdk/aws-apigateway.RestApi.addModel
+removed:@aws-cdk/aws-ecs.BaseServiceOptions.propagateTags
```

Comment:
I'm sorry, I'm a little confused about what this PR is doing. Are we renaming `propagateTags` to `propagateTaskTagsFrom`?

Reply:
There's already a property `propagateTaskTagsFrom` that exists in the FargateServiceProps and the Ec2ServiceProps. The 'propagateTags' existed in the BaseServiceOptions, which is extended by both of those props. 

I moved `propagateTags` into the BaseServiceProps so that it's still used by BaseService, but for customers using FargateServiceProps and Ec2ServiceProps, they currently see both `propagateTags` and `propagateTaskTagsFrom`, but `propagateTags` is not actually used, and if they do set it, it's overridden with the value from `propagateTaskTagsFrom`